app.js

```diff
@@ -1,5 +1,3 @@
-// import { DateTime } from "luxon";
-
 const booksContainer = document.getElementById('books');
```

Comment:
- [ ] It would be better if our README file is descriptive, currently it is not, we have the recommended template but we have to personalized the README to be descriptive. For example; we should have a descriptive title instead of `Project Name` and also since this project is not a collaborative one we should correct it as well